Makefile

```diff
@@ -1,8 +0,0 @@
-buildit:
```

Comment:
I would rather keep the Makefile around if possible, it's a pretty nice and universal way of encapsulating special build/test processes.